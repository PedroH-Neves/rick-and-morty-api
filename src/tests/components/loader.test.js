import React from 'react';
import { render } from '@testing-library/react';
import Loader from '../../components/Loader';

it('Renders Loader', () => {
  const loader = render;

  render(<Loader />);

  expect(loader).toMatchSnapshot();
});
