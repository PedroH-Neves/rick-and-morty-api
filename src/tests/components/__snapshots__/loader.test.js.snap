// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Loader 1`] = `[Function]`;
